domain-models-runtime/src/test/java/org/folio/rest/persist/PostgresClientIT.java

```diff
@@ -2294,14 +2302,23 @@ public void executeConnectionThrowsException(TestContext context) throws Excepti
 
   @Test
   public void executeParam(TestContext context) {
-    Async async = context.async();
     JsonArray ids = new JsonArray().add(randomUuid()).add(randomUuid());
     insertXAndSingleQuotePojo(context, ids)
-    .execute("DELETE FROM tenant_raml_module_builder.foo WHERE id=$1", Tuple.of(UUID.fromString(ids.getString(0))), res -> {
-      assertSuccess(context, res);
-      context.assertEquals(1, res.result().rowCount());
-      async.complete();
-    });
+        .execute("DELETE FROM tenant_raml_module_builder.foo WHERE id=$1", Tuple.of(UUID.fromString(ids.getString(0))), res -> {
+          assertSuccess(context, res);
+          context.assertEquals(1, res.result().rowCount());
+        });
+  }
+
+  @Test
+  public void executeParamFuture(TestContext context) {
+    JsonArray ids = new JsonArray().add(randomUuid()).add(randomUuid());
+    insertXAndSingleQuotePojo(context, ids)
+        .execute("DELETE FROM tenant_raml_module_builder.foo WHERE id=$1", Tuple.of(UUID.fromString(ids.getString(0))))
+        .onComplete(res -> {
+          assertSuccess(context, res);
+          context.assertEquals(1, res.result().rowCount());
```

Comment:
The test ends before these assert are executed.
Either use async=context.async() + async.complete(), or use context.asyncAssertSuccess.

Reply:
yes